### PR TITLE
San 3039 syslog

### PR DESCRIPTION
**Pull request by cflynn07**
Removed all console-based logging and replaced with bunyan logging and a stream to syslogd.

- [x] tested on gamma

- [x] @anandkumarpatel
- [ ] @bkendall

### CODE REVIEW

**Comment by forrestj** [comment]
@cflynn07 @und1sk0  what are the next steps?

**Comment by bkendall** [comment]
@und1sk0 @cflynn07 this is missing the `ip` module in `package.json`

**Comment by cflynn07** [comment]
@forrestj Unsure. Need @und1sk0 to tell me results of his tests with these changes.

**Comment by anandkumarpatel** [comment]
@cflynn07 test failing

**Comment by cflynn07** [comment]
@anandkumarpatel Tests are passing

File: drivers/dockerExecDriver.js [hunk 1]
```diff
@@ -1,14 +1,23 @@
...
+var dockerPath;
 var eventEmitter = new events.EventEmitter();
 var found = false;
 
 which('docker', function(err, cmdpath) {
   if (err) {
-    console.log('err finding docker '+err);
+    log.trace({
+      err: err
+    }, 'err finding docker');
```
Comment (bkendall) [inline drivers/dockerExecDriver.js]: `log.fatal`

File: drivers/nsenterDriver.js [hunk 2]
```diff
@@ -1,14 +1,23 @@
...
 var eventEmitter = new events.EventEmitter();
 var found = false;
+var nsenterPath;
 
 which('docker-enter', function(err, cmdpath) {
   if (err) {
-    console.log('err with docker-enter: '+err);
+    log.trace({
+      err: err
+    }, 'err with docker-enter');
```
Comment (bkendall) [inline drivers/nsenterDriver.js]: `log.fatal`

**Pull request merged**